commit 73e9e18ea09ffffcaac50237def0d9728a213c02
Author: Scott Talbert <swt@techie.net>
Date:   Sat Feb 20 00:08:14 2016 -0500

    Fix STC compilation with GCC6
    
    Use std::abs() from <cmath> instead of abs() from <math.h> to avoid problems
    with ambiguous overloads.
    
    Closes #17147.
    
    Closes https://github.com/wxWidgets/wxWidgets/pull/222

diff --git a/src/stc/scintilla/src/Editor.cxx b/src/stc/scintilla/src/Editor.cxx
index cd72953a..2081df28 100644
--- a/src/stc/scintilla/src/Editor.cxx
+++ b/src/stc/scintilla/src/Editor.cxx
@@ -11,6 +11,7 @@
 #include <ctype.h>
 #include <assert.h>
 
+#include <cmath>
 #include <string>
 #include <vector>
 #include <map>
@@ -5841,9 +5842,9 @@ void Editor::GoToLine(int lineNo) {
 }
 
 static bool Close(Point pt1, Point pt2) {
-	if (abs(pt1.x - pt2.x) > 3)
+	if (std::abs(pt1.x - pt2.x) > 3)
 		return false;
-	if (abs(pt1.y - pt2.y) > 3)
+	if (std::abs(pt1.y - pt2.y) > 3)
 		return false;
 	return true;
 }
